### DISCUSSION TITLE
sent

### DISCUSSION THREAD

**Discussion by ghost** [catppuccin/catppuccin#2389] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Via `static const char *colors[]` in `config.h`

### Any additional comments?

https://tools.suckless.org/sent/

I am more than willing to do this myself. I just want a thumbs up that this is something that is ok to be made. I'm not too sure if source mods are accepted for ports. Otherwise I can get this done tonight. [Licensing is not an issue](https://git.suckless.org/sent/file/LICENSE.html)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by ghost** [comment]
Was told to give it a shot in the Discord. Patch set coming tonight

**Comment by ghost** [comment]
First draft is up https://github.com/xzi/sent/tree/main/patches

**Comment by sgoudham** [comment]
Closing as further discussion should take in place in #2390
